### ISSUE TITLE
Asterisks in indented code blocks parsed as Markdown emphasis

### ISSUE THREAD

**Issue by aroben** [atom/language-gfm#70]
Given Markdown such as the following:

```markdown
The following code shows complete example how to use styleguide with gulp-sass and with gulp watch.

    var styleguide = require('sc5-styleguide');
    var sass = require('gulp-sass'),
    var outputPath = 'output';

    gulp.task('styleguide:generate', function() {
      return gulp.src('*.scss')
        .pipe(styleguide.generate({
            title: 'My Styleguide',
            server: true,
            rootPath: outputPath,
            overviewPath: 'README.md',
            styleVariables: 'lib/app/sass/_styleguide_variables.scss'
          }))
        .pipe(gulp.dest(outputPath));
    });
```

The asterisk in `*.scss` gets parsed as a Markdown emphasis character, causing the rest of the code block to appear in italics. It looks like the grammar doesn't recognize indented code blocks at all (instead only recognizing fenced code blocks).

This bug isn't visible in the Atom Dark or Atom Light themes, but is visible in the Base 16 Tomorrow themes.

**Comment by aroben** [comment]
The bug is also visible on `github.com`, which uses this grammar for highlighting Markdown files.

**Comment by 50Wliu** [comment]
This seems like a duplicate of #44, right?

**Comment by kevinsawicki** [comment]
@50Wliu I believe so, thanks, closing.

**Comment by aroben** [comment]
Maybe it is a duplicate, though the issues seem a little different to me. It seems to me that inside indented code blocks we shouldn't be parsing Markdown syntax (asterisks, underscores, heading markers, etc.) at all. That feels like a separate issue from the behavior of a single unmatched asterisk in normal text. But I don't know much about how this grammar works, so maybe they're more connected than they seem.

**Comment by kevinsawicki** [comment]
Yeah, sorry, indented code blocks are not currently supported at all.

I've updated the title to make that clearer.  Since they aren't supported, they are parsed "regularly" so italic, bold, emoji, etc. will show up in them currently.

This was attempted once but was not sucessful, https://github.com/atom/language-gfm/pull/4

**Comment by gageas** [comment]
Hi, I noticed that [asterisk or underline followed by any character but space] in 4 space indented code block breaks syntax highlighting.
(Actual condition is more complicated. adding symbols or character around asterisk leads some change in highlighting.)
Is this same problem?

![default](https://user-images.githubusercontent.com/40446/30507760-d559c08c-9ac3-11e7-87ef-70a8ed16b2b7.png)
title3 is not highlighted correctly, rendering preview is fine.

**Comment by wolftune** [comment]
Since this is a GitHub issue too, shouldn't a bug be opened at some GitHub-markdown location? Or does that even exist? I can't find it, but it seems there ought to be such a thing. Is it not a dependency of language-gfm?
